runsc/sandbox/network.go

```diff
@@ -145,6 +145,19 @@ func createInterfacesAndRoutesFromNS(conn *urpc.Client, nsPath string, hardwareG
 			continue
 		}
 
+		// Collect data from the ARP table
+		dump, err := netlink.NeighList(iface.Index, 0)
```

Comment:
This error should be checked.

Reply:
Yup, good catch.
Fixed, but I'm not sure if it should bail out or just continue with a warning.
Netlink go library that gvisor uses passes all kinds of errors there, not all of them are fatal. And some are even retriable. But it's never checked in other netlink calls in that file, so...